### DISCUSSION TITLE
Change 'pagination' to 'partitioning'

### DISCUSSION THREAD

**Discussion by WLefever-Cegeka** [Informatievlaanderen/VSDS-LDESServer4J#730] [General]
In the context of VSDS we would like to replace the term 'pagination' with 'partitioning'.

**Comment by Tomvbe** [comment]
Are we sure that we want this change? This has a substantial impact as it impacts the configuration of all servers, E2E, documentation, etc. That's the same reason we have not removed timebased fragmentation yet. Imo, this would entail a flow of deprecating pagination and introducing partitioning and migrate slowly away from pagination.

**Reply by sandervd** [reply]
Hmmm... For me, fragmentation and partitioning would be considered synonyms, which are distinct from pagination?

**Reply by WLefever-Cegeka** [reply]
With 'we woule like', I actually meant Digitaal Vlaanderen. For me personally, it's the same whether it's called pagination or patitioning.
